cmd/resim/commands/styling.go

```diff
@@ -59,15 +61,82 @@ var ReSimUsageTemplate string = `{{StyleHeading "USAGE"}}{{if .Runnable}}
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `
 
-var templateFuncs = template.FuncMap{
+var FlagSetUsageTemplate string = `{{if .HasRequired}}
+
+{{StyleHeading "REQUIRED FLAGS"}}
+{{.RequiredUsages}}{{end}}{{if .HasOptional}}
+
+{{StyleHeading "OPTIONAL FLAGS"}}
+{{.OptionalUsages}}{{end}}
+`
+
+var flagTemplateFuncs = template.FuncMap{
 	"StyleHeading": styleHeading,
 }
 
+var resimTemplateFuncs = template.FuncMap{
+	"StyleHeading":  styleHeading,
+	"FlagSetUsages": flagSetUsages,
+}
+
+var BashCompOneRequiredFlag string = "cobra_annotation_bash_completion_one_required_flag"
```

Comment:
(nit) might be worth documenting this in a comment?

Reply:
probably this should be a non-exported constant, though? `const bashCompOneRequiredFlag`

Reply:
Thank you. That's what I wanted. Showing my go novice status here.